tests/e2e/base.py

```diff
@@ -37,6 +46,51 @@
 ]
 
 
+class Parameters:
+    param_d: Dict[str, Any]
```

Comment:
Is this needed? I don't see any usage of this as a class attribute - only instance. Maybe we could add a type hint in the constructor?

Reply:
you are right. I always thought of this as type hint for class instance attribute. I guess it makes sense this is type hint but for whole class scope